agent/cmd/determined-agent/run.go

```diff
@@ -145,7 +145,7 @@ func newRunCmd() *cobra.Command {
 		"Addresses that the agent's containers should not proxy")
 
 	// Logging flags.
-	cmd.Flags().StringVar(&opts.Fluent.Image, "fluent-image", "fluent/fluent-bit:1.9.3",
+	cmd.Flags().StringVar(&opts.Fluent.Image, "fluent-image", aproto.FluentImage,
```

Comment:
Just wondering why historically we have ```fluent-image``` as a per image config rather than as a master config option? Is there a use case for different ```fluent-image``` per different agent?

Reply:
static agents don't get this value from master but from their own config. master option is just for the agents spun up by provisioner.
in retrospect this could be driven by master, but I'd rather move fluent into task containers (which is a tech debt project) rather than rework this.